### PR TITLE
feat: Implement post and update news article from the ContentRichEditor - EXO-72081_72082 - Meeds-io/MIPs#128

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java [hunk 1]
```diff
@@ -723,10 +723,12 @@ public News scheduleNews(News news, Identity currentIdentity, String newsObjectT
...
-      if (news.isPublished()) {
-        publishNews(news, currentIdentity.getUserId());
-      } else {
-        unpublishNews(news.getId(), currentIdentity.getUserId());
+      if (NewsUtils.canPublishNews(space.getId(), currentIdentity)) {
+        if (news.isPublished()) {
+          publishNews(news, currentIdentity.getUserId());
+        } else {
+          unpublishNews(news.getId(), currentIdentity.getUserId());
+        }
```
Comment (azayati) [inline content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java:731]: why this is changed ?

File: content-webapp/src/main/webapp/news-activity-composer-app/components/ContentRichEditor.vue [hunk 2]
```diff
@@ -57,12 +58,15 @@
...
       :news-id="articleId"
       :news-type="articleType"
+      :space-id="spaceId"
       @post-article="postArticle" />
   </v-app>
 </template>
 
 <script>
 
+import {getURLQueryParam} from '../main';
```
Comment (azayati) [inline content-webapp/src/main/webapp/news-activity-composer-app/components/ContentRichEditor.vue]: can we avoid this import ?

**Review by azayati — CHANGES_REQUESTED** [review]

File: content-webapp/src/main/webapp/news-activity-composer-app/components/ContentRichEditor.vue (hunk 2 above)
Comment (sofyenne) [reply content-webapp/src/main/webapp/news-activity-composer-app/components/ContentRichEditor.vue]: updated!

File: content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java (hunk 1 above)
Comment (sofyenne) [reply content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java:731]: to handle unauthorized exception when the user is a space redactor and not a space publisher !

File: content-webapp/src/main/webapp/news-activity-composer-app/main.js [hunk 3]
```diff
@@ -94,7 +88,7 @@ export function destroy() {
   }
 }
 
-function getURLQueryParam(paramName) {
+export function getURLQueryParam(paramName) {
```
Comment (azayati) [inline content-webapp/src/main/webapp/news-activity-composer-app/main.js]:
```suggestion
function getURLQueryParam(paramName) {
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]

**Pull request merged**